### DISCUSSION TITLE
新的V1.5版本无法正常抓取影片信息，退回到1.4.2就正常了...

### DISCUSSION THREAD

**Discussion by ghost** [Yuukiy/JavSP#140] [问答 | Q&A] [answered]
如题

**Comment by 1zhouzhou** [comment]


**Reply by 1zhouzhou** [reply]
log里的记录如下：
2023-11-26 16:48:55 main:94 DEBUG: web.javbus: 网络错误，正在重试 (1/3): 
SSLError(MaxRetryError("HTTPSConnectionPool(host='www.busfan.cfd', port=443): Max retries exceeded with url: /AVOP-004 (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))"))
能力有限不知道是不是本地环境问题。
版本1.5，全局梯子（日本IP）

**Comment by Yuukiy** [comment]
有日志么？

**Reply by ghost** [reply]
[log.zip](https://github.com/Yuukiy/JavSP/files/13507478/log.zip)

**Comment by Yuukiy** [comment, marked as answer]
> 

试一下新的1.5.1，还有这个问题么？

**Comment by ghost** [comment]
可以了，没问题了
发自我的 iPhone

在 2023年12月2日，21:06，Yuukiy ***@***.***> 写道：

﻿

试一下新的1.5.1，还有这个问题么？
